### PR TITLE
Bump to LLVM green commit 4546397e39589f0a6a707218349d1bf65fe54645 from Oct. 17

### PR DESCRIPTION
**Pull request by gargaroff**

### CODE REVIEW

File: .buildbot/Jenkinsfile [hunk 1]
```diff
@@ -90,7 +90,7 @@ def call() {
       MODELZOO_REPORTDIR    = 'modelzoo_report'
       MODELZOO_HTML         = 'modelzoo.html'
       MODELZOO_STDOUT       = 'modelzoo.stdout'
-      MODELZOO_PUBLISH      = "${JENKINS_HOME}/jobs/${JOB_NAME}/htmlreports/Model_20Zoo_20Report"
+      MODELZOO_PUBLISHDIR   = "${JENKINS_HOME}/jobs/${JOB_NAME}/htmlreports/Model_20Zoo_20Report"
```
Comment (ljfitz) [inline .buildbot/Jenkinsfile:93]: As discussed, this and some other changed files seem to be from a faulty merge. I checked for expected differences using:
```
git difftool -d `git merge-base origin/main HEAD` HEAD --
```
and:
```
(for i in  $(git diff --name-only `git merge-base origin/main origin/feature/onnx_to_torch` origin/feature/onnx_to_torch); do git diff `git merge-base origin/main origin/feature/onnx_to_torch` origin/feature/onnx_to_torch -- $i >old; git diff `git merge-base origin/main HEAD` HEAD -- $i >new; echo "### $i"; xxdiff -D old new; done)
```

Comment (ljfitz) [inline .buildbot/Jenkinsfile:93]: As discussed, this and some other changed files seem to be from a faulty merge. I checked for expected differences using:
```
git difftool -d `git merge-base origin/main HEAD` HEAD --
```
and:
```
(for i in  $(git diff --name-only `git merge-base origin/main origin/feature/onnx_to_torch` origin/feature/onnx_to_torch); do git diff `git merge-base origin/main origin/feature/onnx_to_torch` origin/feature/onnx_to_torch -- $i >old; git diff `git merge-base origin/main HEAD` HEAD -- $i >new; echo "### $i"; xxdiff -D old new; done)
```

**Review by ljfitz — CHANGES_REQUESTED** [review]

**Comment by gargaroff** [comment]
Subsumed by #86

File: .buildbot/Jenkinsfile (hunk 1 above)
Comment (gargaroff) [reply .buildbot/Jenkinsfile:93]: These changes still show up, but if you check https://github.com/ljfitz/onnx-mlir/tree/476e2b8f436feb12592759c0a0deb4987baa1c42/.buildbot (i.e. the exact commit hash that I'm merging into `feature/onnx_to_torch`, the changes are identical

**Pull request closed without merging**